test/jdk/com/sun/management/UnixOperatingSystemMXBean/GetMaxFileDescriptorCount.java

```diff
@@ -22,10 +22,13 @@
  */
 
 /*
- *
+ * @test
  * @bug     4858522
  * @summary Basic unit test of UnixOperatingSystemMXBean.getMaxFileDescriptorCount()
  * @author  Steve Bohne
+ * @requires (os.family=="linux")
```

Comment:
please add spaces, braces are not required, see examples in jtreg tests
@requires os.family == "linux"